### PR TITLE
added cloud v local password manager + keepassx

### PR DESCRIPTION
**Pull request by mfc**
from: https://github.com/cutealism/so-you-want-to-fight-the-state/issues/1

### CODE REVIEW

File: src/data/password-manager.json [hunk 1]
```diff
@@ -17,15 +17,28 @@
...
             },
             {
                 "prompt": "Still seems to hard.",
                 "step": "aww"
             }
         ]
     },
-    "install": {
+    "where": {
+        "description": "Do you wanna install the password manager in your browser and have it sync with your phone for easy access, or only be able to access it locally your computer?",
```
Comment (imightbeamy) [inline src/data/password-manager.json]: Could you change the wording there? I like the style be more like "Here are the two options, which do you like?" 

Maybe something like:
> There are a few options for password managers! You could install one in your browser and have it sync with your phone for easy access, or keep your passwords extra safe by storing them locally your computer.

File: src/data/password-manager.json [hunk 2]
```diff
@@ -38,6 +51,19 @@
             }
         ]
     },
+    "install-local": {
+        "description": "<a href='https://www.keepassx.org/downloads'>KeePassX</a> is a free password manager for Windows, Mac, and Linux! It's got that 90s aesthetic that may be cool again at some point.",
```
Comment (imightbeamy) [inline src/data/password-manager.json:55]: This is cute 😊

**Review by imightbeamy — COMMENTED** [review]
Thanks! I'm fine adding this as an option, but want the copy to be a little more even handed about the two options.

**Comment by imightbeamy** [comment]
I'll deploy this later today or tomorrow! Thanks for contributing :)

bitmoji

**Comment by imightbeamy** [comment]
This has been deployed! Thanks again!

**Comment by mfc** [comment]
sweet, thanks for creating such a nice resource!

**Pull request merged**